index.css

```diff
@@ -1,3 +1,39 @@
-html {
-  
+@import url(https://fonts.googleapis.com/css?family=Alegreya:400,900);
+
+h1 {
+	margin: 50px auto;
+	font-size: 50px;
+}
+
+body {
+  font-family: 'Alegreya', serif;
+  text-align: center;
+}
+
+table {
+  margin-left: auto;
+  margin-right: auto;
+}
+
+  .square{
+      width:100px;
+      height:100px;
+  }
+
+  .v{
```

Comment:
I feel like this class and the `h` class could be named to be more contextual about what it is used for.